Fix Conversions to show °C and °F

ilv opened the pull request:
Fix to #827. I hope it is what you were looking for. 

I also modified the tests file, although there are some tests not related to temperatures that do not pass. Two of them don't pass because of the words mi and sq instead of miles and square. The rest fail because of precision.

MrChrisW commented:
@ilv Thanks for your contribution, after a quick look this looks great :+1: Will await sign off from @mintsoft & @jagtalon

mintsoft commented:
@ilv can you add tests to ensure that it still handles queries such as "40 degrees celsius in fahrenheit" as we want to support both :+1: 

The output format can be like you've made it, it's just the inputs that need testing.

mintsoft commented:
@ilv I think you may be working on an old version of the C::P module too judging bby your comments, have you tried `cpanm Convert::Pluggible` ?

jagtalon commented:
@ilv nice! Agree with @mintsoft would like to see if "40 degrees celsius in fahrenheit" is working.

jagtalon commented:
Oh hang on I misread it--I thought it accepts queries such as "40 degrees celsius in °F". @mintsoft i guess it doesn't change what it accepts as input so should be good! :+1:

mintsoft commented:
@jagtalon As did I actually! Re-reading it's not changed the input behaviour at all

ilv commented:
Indeed, I did not change the input behaviour, I just implemented what was proposed in #827  :)

@mintsoft Yeah, I was using the version installed on the Virtual Machine from 2014-12-23. I updated Convert::Pluggable and now all test passes! :+1:

mintsoft commented:
@ilv Cool! Good to hear. Thanks for the work on this
